samples/test/util.js

```diff
@@ -0,0 +1,90 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const uuid = require('uuid');
+const compute = require('@google-cloud/compute');
+
+const PREFIX = 'gcloud-test-';
+
+const instancesClient = new compute.InstancesClient();
+
+// Get a unique ID to use for test resources.
+function generateTestId() {
+  return `${PREFIX}-${uuid.v4()}`.substr(0, 30);
+}
+
+/**
+ * Get VM instances created more than one hour ago.
+ */
+async function getStaleVMInstances() {
+  const projectId = await instancesClient.getProjectId();
+  const result = [];
+  const currentDate = new Date();
+  currentDate.setHours(currentDate.getHours() - 1);
```

Comment:
Curious why minor 1 hour here?

Reply:
Good point, for compute tests I switched to 3 hours.

Originally I got idea from nodejs-bigtable, there we have 1 hour: https://github.com/googleapis/nodejs-bigtable/blob/80022f846081a67c610870cec0e26f1da76d0dd5/samples/test/util.js#L44

Reply:
Not sure the purpose. But except for this, in generall, looks good to me.